game-core/src/main/java/games/strategy/util/Version.java

```diff
@@ -35,17 +34,9 @@ public Version(final int major, final int minor) {
    * Constructs a Version object without the micro version, defaults to 0.
```

Comment:
This Javadoc is no longer applicable.  The one below that was removed would probably be more appropriate.

There's a few other comments within this file that still make reference to the four-component form.  Those should probably be updated to reflect the current state.